src/mtev_websocket_client.c

```diff
@@ -316,7 +319,7 @@ mtev_websocket_client_drive(eventer_t e, int mask, void *closure, struct timeval
 
 mtev_websocket_client_t *
 mtev_websocket_client_new(const char *host, int port, const char *path, const char *service,
-                          mtev_websocket_client_callbacks *callbacks) {
+                          mtev_websocket_client_callbacks *callbacks, void *closure, const char *pool_name) {
```

Comment:
It's a nit, but this has it assigned to the default pool if someone typos the name.  That seems like it might be better to control at the callsight.  What about changing this to take an `eventer_pool_t *` and requiring the caller to do the lookup (and thus handle a failed lookup)?